openerp/api.py

```diff
@@ -404,30 +404,79 @@ def old_api(self, cr, uid, ids, *args, **kwargs):
 
 def one(method):
     """ Decorate a record-style method where ``self`` is expected to be a
-        singleton instance. The decorated method automatically loops on records,
-        and makes a list with the results. In case the method is decorated with
-        :func:`returns`, it concatenates the resulting instances. Such a
-        method::
+        singleton instance. The decorated method automatically loops on
+        records::
 
             @api.one
-            def method(self, args):
-                return self.name
+            def rename(self, name):
+                self.name = name
 
-        may be called in both record and traditional styles, like::
+        The method may be called in both record and traditional styles, like::
 
             # recs = model.browse(cr, uid, ids, context)
-            names = recs.method(args)
-
-            names = model.method(cr, uid, ids, args, context=context)
+            names = recs.rename(args)
```

Comment:
``rename()`` does not return anything here, affecting ``names`` is useless I think